sklearn/metrics/pairwise.py

```diff
@@ -19,6 +19,8 @@
 from scipy.sparse import csr_matrix, issparse
 from scipy.spatial import distance
 
+from sklearn.utils import _array_api
```

Comment:
for consistency we could add `_clip` in the `from ..utils._array_api import ...` below

Reply:
Done, thanks!